src/lib/gql/zeusHasuraAdmin/index.ts

```diff
@@ -16240,6 +16240,9 @@ const handleFetchResponse = (
 export const apiFetch =
   (options: fetchOptions) =>
   (query: string, variables: Record<string, any> = {}) => {
+    if (!!process.env.NODE_ENV) {
+      globalThis.fetch = nodeFetch as unknown as typeof fetch;
```

Comment:
how about splitting this to a separate PR ?

Reply:
Sure why not.

Reply:
that will be great thanks! will def help unblock others